bitnami/cassandra/Chart.yaml

```
@@ -23,4 +23,4 @@ name: cassandra
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/cassandra
   - http://cassandra.apache.org
-version: 9.6.1
+version: 9.7.1
```

Thanks for the PR, please note there is a typo when bumping the version
```suggestion
version: 9.7.0
```

---

Ouch... fixed in 39e5ea4